ddtrace/vendor/__init__.py

```diff
@@ -102,3 +117,11 @@
 
 
 base.log = get_logger('ddtrace.vendor.dogstatsd')
+
+
+if sys.version_info >= (3, 8, 0):
+  import bytecode0130 as bytecode
+elif sys.version_info >= (3, 7, 0):
+  import bytecode0141 as bytecode
+
+sys.modules["ddtrace.vendor.bytecode"] = bytecode
```

Comment:
```suggestion
```

Reply:
or perhaps do
~~~python
sys.modules["bytecode"] = bytecode
~~~
to allow Python versions for which we don't vendor to work as expected with the same module path?